src/components/SourceTabs.js

```diff
@@ -182,6 +196,10 @@ const SourceTabs = React.createClass({
       items.push({ item: showSourceMenuItem });
     }
 
+    if (isEnabled("prettyPrint")) {
+      items.push({ item: prettyPrint });
+    }
```

Comment:
hmm, i think its safe to assume `prettyPrint` is on :)

are we using the flag in other places? If so we should rip them out too